config/l5-swagger.php

```diff
@@ -8,6 +8,11 @@
                 'title' => 'L5 Swagger UI',
             ],
 
+            /*
+             * Use your own generated json documentation file
+            */
+            'use_own_docs' => false,
```

Comment:
We do have `generate_always` config param https://github.com/DarkaOnLine/L5-Swagger/blob/master/config/l5-swagger.php#L223 which can be used to disable generation on UI load

Reply:
I have used the `generate_always `config to prevent doc generation on the console command on https://github.com/DarkaOnLine/L5-Swagger/pull/584/files. Better we can introduce a new config to control the console commands